features/dom.yml

```diff
@@ -123,6 +124,8 @@ compat_features:
   - api.Document.replaceChildren
   - api.Document.scripts
   - api.Document.title
+  - api.Document.write
```

Comment:
Drop this key please. It should've been marked as deprecated. I've sent https://github.com/mdn/browser-compat-data/pull/25468